tests/oldcheck/check_test.go

```diff
@@ -100,9 +100,6 @@ func runTest(t *testing.T, client pb.OpenFGAServiceClient, tests checkTests) {
 				})
 				require.NoError(t, err)
 				require.Equal(t, assertion.Expectation, resp.Allowed, assertion)
-				if assertion.Trace != "" {
```

Comment:
Removed this since we're moving away from the `resolution` field and the refactor doesn't have support for the trace resolution string. I can add the trace resolution string easily, but since we're moving away from it I decided to omit that from the scope of this PR.